pom.xml

```diff
@@ -39,6 +39,11 @@
     <dependencies>
         <dependency>
             <groupId>com.github.ONSdigital</groupId>
+            <artifactId>dp-dd-request-filter</artifactId>
+            <version>v1.0.0</version>
```

Comment:
Should probably use v1.0.1. The only difference is the readme and example config, admittedly.